### PR TITLE
Turf inside activity

### PR DESCRIPTION
**Pull request by langsmith**
👀 @cammace

This branch adds a Mapbox Java Turf example to the [Mapbox Android Services](https://github.com/mapbox/mapbox-java/tree/19450d5cefb8feaf159226eec81f32befad697c3) section of the demo app. It uses the TurfJoins.inside() method to determine whether the marker location is within the polygon area. 

[For more info about Turf and its geospatial analysis functionality](https://www.mapbox.com/analysis/)

giphy

screenshot

### CODE REVIEW

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/mas/TurfInsideActivity.java [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    mapView.onDestroy();
+  }
+
+  @Override
+  protected void onSaveInstanceState(Bundle outState) {
+    super.onSaveInstanceState(outState);
+    mapView.onSaveInstanceState(outState);
+  }
+
+  public void onMapClick() {
```
Comment (cammace) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/mas/TurfInsideActivity.java]: Looks like this method only handles the onclick? in this case you can just add it as an implementation like so:

```java
... extends AppCompatActivity implements MapboxMap.OnMapClickListener {
```

```
 @Override
  public void onMapReady(MapboxMap mapboxMap) {
    mapboxMap.setOnMapClickListener(this);
```

```
@Override
  public void onMapClick(@NonNull LatLng point) {
}
```

File: MapboxAndroidDemo/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -81,6 +81,7 @@
     <string name="activity_location_tracking_title">Track user location</string>
     <string name="activity_location_animated_icon_title">Animate the user icon</string>
     <string name="activity_location_customize_user_title">Customize user location icon</string>
+    <string name="activity_location_turf_inside_title">Use Java Turf</string>
```
Comment (cammace) [inline MapboxAndroidDemo/src/main/res/values/strings.xml]: Can we name the title something along the lines of `turf inside`. Java Turf is too generic for this example

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/mas/TurfInsideActivity.java (hunk 1 above)
Comment (langsmith) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/mas/TurfInsideActivity.java]: Cool

Comment (langsmith) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/mas/TurfInsideActivity.java]: Just doing (this) throws this error:

**Pull request merged**